Device_Profile_Sample/Device_Profile_Get_Identity_Data/Program.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Microsoft.IdentityModel.Clients.ActiveDirectory;
```

Comment:
I removed all of the bloat code and call an api endpoint which prints team project info, so it should match the other samples.